articles/aks/web-app-routing.md

```diff
@@ -327,9 +329,6 @@ spec:
     tls:
       skipClientCertValidation: false
   sources:
-  - kind: Service
```

Comment:
This is still useful/needed as it limits the allowed IP addresses to the endpoints behind the Service

Reply:
Got it. Just realized that we are adding app-routing-system namespace to OSM with sidecar-injection disabled. Thanks Keith.